WalletWasabi.Fluent/ViewModels/Wallets/Advanced/WalletCoins/WalletCoinsViewModel.cs

```diff
@@ -55,8 +55,8 @@ protected override void OnNavigatedTo(bool isInHistory, CompositeDisposable disp
 		// [Column]			[View]					[Header]	[Width]		[MinWidth]		[MaxWidth]	[CanUserSort]
 		// Indicators		IndicatorsColumnView	-			Auto		-				-			false
 		// Amount			AmountColumnView		Amount		Auto		-				-			true
-		// AnonymitySet		AnonymityColumnView		<custom>	40			-				-			true
-		// Labels			LabelsColumnView		Labels		*			-				-			false
+		// AnonymityScore	AnonymityColumnView		<custom>	50			-				490			true
```

Comment:
490 should go the the `Labels` row, right?
As far as I can see the maxwidth was set on the Labels column.

Reply:
Fixed